contribution/mishba-ai.mdx

```diff
@@ -0,0 +1,42 @@
+---
+author: misba bagban
+github_username: mishba-ai
+occupation: Student
+---
+
+# Introduction
+
+Hi I'm misba bagban.I am currently a 1st year Computer engineering student persuing B-Tech.
+I love to contributing to OpenSource and getting involved in awesome tech communities.
+
+## Skills
+
+<Skills>
+  <Skill
+    name="html"
+   icon={
+   <svg xmlns="http://www.w3.org/2000/svg" class="icon icon-tabler icon-tabler-brand-html5" width="24" height="24" viewBox="0 0 24 24" stroke-width="2" stroke="currentColor" fill="none" stroke-linecap="round" stroke-linejoin="round">
+   <title>html</title>
+   <path stroke="none" d="M0 0h24v24H0z" fill="none"></path>
+   <path d="M20 4l-2 14.5l-6 2l-6 -2l-2 -14.5z"></path>
+   <path d="M15.5 8h-7l.5 4h6l-.5 3.5l-2.5 .75l-2.5 -.75l-.1 -.5"></path>
+</svg>
+}
+  >
+   Html
```

Comment:
Everything looks good. But could you add your experience such as `1-year experience with HTML` 

``` suggestion
1-year experience with HTML
```